cmd/ore/oci/create.go

```diff
@@ -46,30 +46,51 @@ func runCreateVCN(cmd *cobra.Command, args []string) {
 		os.Exit(1)
 	}
 
-	vcn, err := API.CreateVCN("kola", "10.0.0.0/16")
+	vcnResp, err := API.CreateVCN("kola", "10.0.0.0/16")
 	if err != nil {
 		fmt.Fprintf(os.Stderr, "Creating VCN: %v\n", err)
 		os.Exit(1)
 	}
+	vcn := vcnResp.Vcn
 
-	secList, err := API.CreateDefaultSecurityList(vcn.ID)
+	if vcn.Id == nil {
+		fmt.Fprintf(os.Stderr, "received virtual cloud network id nil\n")
+		os.Exit(1)
```

Comment:
I'm not generally a fan of `os.Exit()`ing all over the place. Maybe wrap this in another function call that returns an error and handle all the os.Exit() in the caller?

Either that or maybe have a `exitIfNil(variable interface{}, msg string)` to help with the verbosity